review08/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/client/BaggageClientResource.java

```diff
@@ -5,18 +5,22 @@
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.PathParam;
+import org.eclipse.microprofile.health.HealthCheckResponse;
 import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
 
 import java.util.List;
 
 /**
  * Client for baggage service
  */
-@Path("/baggage/passenger")
 @RegisterRestClient(configKey = "baggage-resource")
 public interface BaggageClientResource {
 
     @GET
-    @Path("/{passengerId}")
+    @Path("/baggage/passenger/{passengerId}")
```

Comment:
intreseting change for 8th lecture :) just be careful to remember what you are changing with regards to the previous states of the project.